types/twine-sugarcube/state.d.ts

```diff
@@ -186,6 +210,27 @@ export interface StateAPI {
          */
         has(key: string): boolean;
 
+        /**
+         * Returns an array of the story metadata store's keys.
+         * @since 2.36.0
+         * @example
+         * // Get the metadata store's keys.
+         * var metadataKeys = State.metadata.keys();
+         *
+         * // Iterate over the keys with a `for` loop.
+         * for (var i = 0; i < metadataKeys.length; ++i) {
+         * 	var key = metadataKeys[i];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/state.d.ts;
  Line: 222;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.